test-suite/client/src/data/schema.ts

```diff
@@ -692,7 +692,245 @@ const schema: Schema = {
+                    [RequestHeaderProps.Header]: [
+                      {
+                        label: "Url",
+                        id: "url",
+                        defaultValue: "http://localhost:8081/report",
+                        validators: [isUrl(), isReportUrl()]
+                      },
+                      {
+                        label: "Content Type",
+                        id: "content-type",
+                        defaultValue: "application/xml",
+                        validators: [notNull(), minLength(7)]
+                      }
+                    ],
+                    [RequestHeaderProps.Body]: [
+                      {
+                        label: "NHS Number",
+                        id: "nhs-number",
+                        defaultValue: "1234567891",
+                        validators: [notNull(), isNumeric(), isLength(10)]
+                      },
+                      {
+                        label: "Recipient ODS Code",
+                        id: "ods-code",
+                        defaultValue: "E88122002",
+                        validators: [notNull(), isAlphanumeric()]
+                      },
+                      {
+                        label: "Recipient DOS Code",
+                        id: "dos-code",
+                        defaultValue: "159744",
+                        validators: [isAlphanumeric()]
+                      },
+                      {
+                        label: "Recipient Name",
+                        id: "recipient-name",
+                        defaultValue: "Bramley Road Surgery",
+                        validators: [notNull(), isAlpha()]
+                      },
+                      {
+                        label: "Recipient Address Line 1",
+                        id: "recipient-street-address-line-1",
+                        defaultValue: "Bramley Road Surgery",
+                        validators: [notNull(), isAlphanumeric()]
+                      },
+                      {
+                        label: "Recipient Address Line 2",
+                        id: "recipient-street-address-line-2",
+                        defaultValue: "2 Bramley Road",
+                        validators: [notNull(), isAlphanumeric()]
+                      },
+                      {
+                        label: "Recipient Town",
+                        id: "recipient-street-address-town",
+                        defaultValue: "Ealing",
+                        validators: [isAlpha()]
+                      },
+                      {
+                        label: "Recipient City",
+                        id: "recipient-street-address-city",
+                        defaultValue: "London",
+                        validators: [isAlpha()]
+                      },
+                      {
+                        label: "Recipient Postcode",
+                        id: "recipient-street-address-postcode",
```

Comment:
Does not appear in ITK message.

Could you update the ITK message with the missing field, other ITK messages should have the field within them... syntax should be along the lines of @@id@@.